Hw for lesson 3

aliyashakirova opened the pull request.

aliyashakirova committed:
Hw for lesson 3

VitalyOv commented on Program.cs:
```diff
@@ -1,9 +1,139 @@
+//            Console.WriteLine("Can't be devided by 0");
+//        }
+//        break;
+//}
```
нужно добавить секцию default на случай если ввели операцию которой нет

aliyashakirova replied:
добавила

VitalyOv commented on Program.cs:
```diff
@@ -1,9 +1,139 @@
+//        {
+//            Console.WriteLine($"The number = {number} is within the range [36..50]");
+//        }
+//    if (number == 50)
```
Почему вы выделили 50 в отдельный диапазон

aliyashakirova replied:
потому что в таске написано так:
Напишите программу определения, попадает ли указанное пользователем число от 0 до
100 в числовой промежуток [0 - 14] [15 - 35] [36 - **_50][50_** - 100].

VitalyOv commented on Program.cs:
```diff
@@ -1,9 +1,139 @@
+else
+{
+    Console.WriteLine("Odd");
+}
```
Нехватает второго способа.

aliyashakirova replied:
добавила

aliyashakirova committed:
- added the default case to the first task's code
- added Task 4.1 - an alternative way of checking if the entered number is even or odd
